src/utils/components/Textarea.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+
+export default function Textarea({
+  children,
+  onChange,
+  className,
+  name,
+  value,
+  rows = 3,
+  label,
+  disabled,
+}: {
+  children: JSX.Element
+  onChange: (e: React.ChangeEvent<HTMLTextAreaElement>) => void
+  className?: string
+  name: string
+  value?: string
+  rows?: number
+  label?: string
+  disabled?: boolean
+}) {
+  return (
+    <div className="flex flex-col">
+      <div className={`flex items-center ${className} `}>
+        <label className="bg-black text-white px-4 py-[32px] border-[3px] border-black rounded-s-lg">
```

Comment:
I suggest using tailwind intelisense extention. It will give us a preview when typing or pressing ctrl+space with valid tailwind classes and a preview for what the class do. For example `py-[32px]`  can be replaced with tailwind design system value like `py-8` which. We will get that this class addes a padding of 32px to the vertical direction from the preview of the extention wihch will make it easier to use tailwind classes instead of using custom or fixed values.